File: exercice2/scala/src/main/scala/fr/devoxx/devops/logs/streaming/SparkStreaming1.scala
package fr.devoxx.devops.logs.streaming

import fr.devoxx.devops.logs.ApacheAccessLog
import org.apache.spark.streaming.StreamingContext

/* Compter le nombre de code http à 404. */
case class SparkStreaming1(hostname: String, port: Int, sc: StreamingContext) {

  def process: Unit = {
    /*
    sc.socketTextStream(hostname, port)
      .xxx()
    */
  }

}

File: exercice2/java/src/main/java/fr/devoxx/devops/logs/streaming/SparkStreaming1.java
package fr.devoxx.devops.logs.streaming;

import fr.devoxx.devops.logs.ApacheAccessLog;
import org.apache.spark.SparkConf;
import org.apache.spark.streaming.Durations;
import org.apache.spark.streaming.api.java.JavaStreamingContext;

import java.io.Serializable;

/**
 * Compter le nombre de code http à 404.
 */
public class SparkStreaming1 implements Serializable {

    public void process(String hostname, int port, JavaStreamingContext sc) {
        /*
        sc.socketTextStream(hostname, port)
                .xxx();
        */
    }

}
